orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -687,11 +696,11 @@ def get_providers():
 
                 providers[providers.index(cuda_ep_name)] = (cuda_ep_name, cuda_ep_options)
 
-            return providers
+            return providers        
```

Comment:
```suggestion
            return providers
```
remove trailing white spaces